pkg/cmd/issue/view/view.go

```diff
@@ -161,22 +196,102 @@ func printHumanIssuePreview(io *iostreams.IOStreams, issue *api.Issue) error {
 	}
 
 	// Body
-	if issue.Body != "" {
-		fmt.Fprintln(out)
-		style := markdown.GetStyle(io.TerminalTheme())
-		md, err := markdown.Render(issue.Body, style, "")
+	fmt.Fprintln(out)
+	if issue.Body == "" {
+		issue.Body = "_No description provided_"
```

Comment:
I opted to add a default issue body when none is provided. This is the default on GitHub.com. I copied the language and styling from there as well. The main reason for adding this is that I felt that for issues without a body but with comments the output looked very odd, and found it difficult to visually parse.